Umount.sh

```diff
@@ -3,8 +3,10 @@
 # Script to clean up all devices mounted under $CHROOT
 #
 #################################################################
+CHROOT=/mnt/ec2-root
```

Comment:
Ahh, I think I see. This has always been set as an environment variable when calling the script. Other scripts in this project have been better about setting the default, [for example](https://github.com/plus3it/AMIgen7/blob/e9e8b505a834fbb6aebae07fc9e47eab87884c09/CleanChroot.sh#L7).

 ```suggestion
CHROOT=${CHROOT:-/mnt/ec2-root}
```